src/community/features-templating/features-templating-core/src/main/java/org/geoserver/featurestemplating/configuration/SupportedFormat.java

```diff
@@ -23,11 +24,10 @@ public String getFormat() {
     }
 
     public static List<SupportedFormat> getByExtension(String extensions) {
```

Comment:
The argument should perhaps be `extension` (singular)

Reply:
fixed